### PR TITLE
Update docs to talk about per pipeline settings removed

### PR DESCRIPTION
**Pull request by ozdenyilmaz**
We got a few customers that asked us about the per pipeline retention settings. So adding it to the docs so it is more clear

https://linear.app/buildkite/issue/PDP-1497/update-docs-to-talk-about-per-pipeline-settings-removed

### CODE REVIEW

**Comment by eleanorakh** [comment]
Thanks Oz! This update reflects the current state of build retention settings ✅ on my part but i'll deff defer to Michael for docs review ✨

**Comment by eleanorakh** [comment]
heya @mbelton-buildkite I'd love your review on this one 🙏🏻

File: pages/pipelines/build_retention.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ The following diagram shows the lifecycle of build data by plan.
   </tbody>
 </table>
 
+Per pipeline retention settings have been retired and all pipelines are now in organisation specific retention settings specified above.
```
Comment (mbelton-buildkite) [inline pages/pipelines/build_retention.md:53]: This currently reads like a changelog, but the docs should reflect the current state. How about:

```suggestion
Retention periods are set according to an organization's plan, as shown in the previous table. Per-pipeline retention settings are not supported.
```

Does that work?

**Comment by eleanorakh** [comment]
Excellent suggestion Michael! I've applied it 🙏🏻

**Review by mbelton-buildkite — APPROVED** [review]
LGTM. Feel free to merge when the build passes 😊

**Pull request merged**